CHANGELOG.rst

```diff
@@ -4,8 +4,8 @@ Changelog
 
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
-1.1.0-SNAPSHOT (2020-06-07)
----------------------------
+1.1.0 (2021-08-02)
```

Comment:
Do we want to have the snapshot also here?

Reply:
? I don't have this section locally

Reply:
![image](https://user-images.githubusercontent.com/11536497/128480104-5e84e293-e8ac-41dc-a70f-33c61cbf8089.png)

Reply:
No, I think we decided against that.

Reply:
I know, I am asking if we want to add it here too.. but I guess it makes not really a difference

Reply:
no , since a release is never a development snapshot in our case, but references a stable commit from `master/main`. I would reference the next stable release goal in the CL.